Docs update (#73)
solidity-labs-io/forge-proposal-simulator: docs/type-check/example.md

# Example Type Check

> After adding FPS to the project dependencies and setting up through the [Setup guide](./introduction.md#setting-up), it's time to go through an example contract to understand how type checking works.

Reviewer: ```suggestion
After adding FPS to the project dependencies and following the [setup guide](./introduction.md#setting-up), it's time to go through an example contract to understand how type checking works.
```

}
```

> The contract constructor arguments appear complex for this example; luckily, we already have the constructor arguments ready for you.

Reviewer: ```suggestion
The contract constructor arguments appear complex for this example; luckily, we already have the constructor arguments ready for you.
```
```suggestion
The contract constructor arguments appear complex for this example; luckily, we already have the constructor arguments ready.
```

}
```

Note: The default artifact folder is `out`. If it is different in your project in `foundry.toml`, change the above artifact path accordingly.

> Now, add the following object in `Addresses.json`:

Reviewer: ```suggestion
Now, add the following object to `Addresses.json`:
```

```json
{
    "addr": "0x293cad26033577eb68137603a34d2dbfd05104d8",
    "chainId": 11155111,
    "name": "ExampleTypeCheck",
    "isContract": true
}
```

The above contract is already deployed on the Sepolia testnet, and it will check that the correct contract is deployed by verifying the bytecode of the Sepolia contract with the local bytecode of the `ExampleTypeCheck` contract.

> Assuming npm packages are already installed and environment variables are added in the [Setup guide](./introduction.md#setting-up).

Reviewer: ```suggestion
Check that npm packages are already installed and environment variables are added. If they aren't, see the [setup guide](./introduction.md#setting-up) for instructions.
```
